bc_obps/reporting/tests/utils/baker_recipes.py

```diff
@@ -84,7 +84,7 @@ def json_seq(
 facility_report = Recipe(
     FacilityReport,
     report_version=foreign_key(report_version),
-    facility=foreign_key(facility),
+    facility=foreign_key(facility, one_to_one=True),
```

Comment:
This is to tell the model bakery not to associate multiple facility reports to the same facility when building it with `_quantity`